Dockerfile

```diff
@@ -96,6 +96,6 @@ CMD source /opt/rh/devtoolset-8/enable && \
     source /opt/rh/rh-ruby27/enable && \
     python3 -m joshua.joshua_agent \
         -C ${FDB_CLUSTER_FILE} \
-        --work_dir /var/joshua \
+        --work_dir /dev/shm/joshua \
```

Comment:
Please make this a parameter, and keep `/var/joshua` as the default value. We size our k8s pods just enough to run simulation tests. We don't have any head room to have work_dir in memory.

Reply:
I agree with @sfc-gh-kmakino . `--workdir` should keep it's current default. Ideally, I'd like to see it pulled up to an ENV that can be substituted at runtime.